### PR TITLE
Add a pet form

### PR DESCRIPTION
**Pull request by annauyenle**
Fixed endpoint in app.js, fixed link in PetTypeContainer to link paths /pets/(type) to /pets/(type)/(id)

### CODE REVIEW

File: src/components/AddAPetForm.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+const AddAPetForm = props => {
+    const [newForm, setNewForm] = useState({})
+    
+    const handleChange = event => {
+        setNewForm({
+            ...newForm,
+            [event.currentTarget.id]: event.currentTarget.value
+        })
+    }
+    const handleSubmit = event => {
```
Comment (prophetsfall) [inline src/components/AddAPetForm.js]: You've defined event handles, but I don't see them being used. Can you clarify?

File: src/components/AddAPetContainer.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React, {useState, useEffect} from "react"
+import AddAPetForm from "./AddAPetForm"
+
+const AddAPetContainer = props => {
+    const [forms, setForms] = useState({})
+
+    const addNewForm = formPayload => {
+        fetch("/api/v1/surrenderedpets", {
```
Comment (prophetsfall) [inline src/components/AddAPetContainer.js]: I don't see an endpoint for this in Server. Just wanted to check if that's intentional or not

File: server/server.js [hunk 3]
```diff
@@ -23,6 +23,7 @@ app.set("view engine", "hbs")
 app.use(logger("dev"))
 app.use(express.json())
 
+
```
Comment (prophetsfall) [inline server/server.js]: please remove extraneous line

File: server/server.js [hunk 4]
```diff
@@ -41,7 +42,7 @@ app.get('/api/v1/pets', (req, res) => {
     })
   })
 
-app.get('/api/pets/:animalType', (req, res) => {
+app.get('/api/v1/pets/:animalType', (req, res) => {
   const petTypeSearch = req.params.animalType
   pool.query('SELECT * from pet_types WHERE type = $1', [petTypeSearch], (error, results) => {
```
Comment (prophetsfall) [inline server/server.js]: looking at your data `type` is a string in your database. Here you have petTypeSearch set to the animal type from req.params, which is a number. 
I think you want to be searching by ID in your query.

File: src/components/AddAPetForm.js (hunk 1 above)
Comment (annauyenle) [reply src/components/AddAPetForm.js]: Sorry, I started this branch to work on this form, but decided to dedicate it to work on fixing the endpoints for the PetType and PetShow containers. Yesterday I had to delete the original branch (i.e. the folder from my computer) which I dedicated to PetType for some config stuff, so I thought it was safer to just do it on a new branch that's merged with the latest version of the master. I though I had deleted AddAPet form and container but it still showed up here.

File: server/server.js (hunk 3 above)
Comment (annauyenle) [reply server/server.js]: Removed

File: src/components/AddAPetContainer.js (hunk 2 above)
Comment (annauyenle) [reply src/components/AddAPetContainer.js]: Please see above comment.

File: server/server.js (hunk 4 above)
Comment (annauyenle) [reply server/server.js]: The directions were a little unclear, our group decided to interpret the instruction to mean that the search parameter would be either "dog" or "cat". That's why petTypeSearch returns a string, we did this intentionally. We intend for the the req.params to be a string of either dog or cat.

**Pull request merged**